tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -528,5 +532,6 @@ module.exports = {
   goToUrl,
   getFastActionItemsLabels,
   getActionBarLabels,
-  getErrorLog
+  getErrorLog,
+  mobileTopBarTitle
```

Comment:
1. Then you don't need to expose `mobileTopBarTitle`.
2. Please add a comma at the end of each item when breaking them in separate lines ([see example](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/#object--array-creation) of the `right` block). Next time someone adds something, it shows 1 line change instead of 2, small thing but practical that we try to do  
```suggestion
  getHeaderTitleOnMobile,
```

Reply:
Done